scripts/breaking_changes_checker/breaking_changes_tracker.py

```diff
@@ -476,38 +477,39 @@ def check_class_instance_attribute_removed_or_renamed(self, components: Dict) ->
                 elif prop.label == "replace":
                     deleted_props = self.stable[self.module_name]["class_nodes"][self.class_name]["properties"]
 
-                for property in deleted_props:
-                    bc = None
-                    if self.class_name.endswith("Client"):
-                        property_type = self.stable[self.module_name]["class_nodes"][self.class_name]["properties"][property]["attr_type"]
-                        if property_type is not None and property_type.lower().endswith("operations"):
-                            bc = (
-                                self.REMOVED_OR_RENAMED_OPERATION_GROUP_MSG,
```

Comment:
The diff got wonky here, but the only thing that should have changed in these lines was the indent.